### DISCUSSION TITLE
How to import data from md / html ?

### DISCUSSION THREAD

**Discussion by se4n01** [docmost/docmost#188] [Help / Questions] [answered]
Hello,
Very nice project, I am wondering if it is possible to import data from other sources in large quantities (10s k of pages) 

Any success stories in this area anyone has to share would be interesting.

Thanks,
Sean

**Comment by Philipinho** [comment, marked as answer]
There is support for Markdown and HTML imports. You can select and upload all the pages.

**Reply by sjwang-01** [reply]
hello。I packaged and imported a Markdown file containing images into Docmost. The file structure is as follows:

xx.md

image/   folder containing all related images    e.g:images/1.png

After importing, the images in the document fail to display. How can this be resolved?
